index.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>YOUR NAME HERE</title>
+  <title>Nathan</title>
 </head>
 <body>
-
+<img src=h"ttps://media.giphy.com/media/NiutNXkkD1ei4/giphy.gif"
```

Comment:
it appears to

On Mon, Mar 14, 2016 at 4:41 PM, Nathan Allen <notifications@github.com>
wrote:

> In index.html
> <https://github.com/sf-wdi-27-28/learn-git/pull/33#discussion_r56095289>:
>
> >  </head>
> >  <body>
> > -
> > +<img src=h"ttps://media.giphy.com/media/NiutNXkkD1ei4/giphy.gif"
>
> Did you check that this image works by opening it in your browser?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sf-wdi-27-28/learn-git/pull/33/files#r56095289>.
>